src/stories/foundations/design_principles.stories.mdx

```diff
@@ -4,7 +4,19 @@ import { Meta } from '@storybook/addon-docs'
 
 # Design Principles
 
+🚧 **WIP!**
+
+## General principles
+
 - **Human Centered:** Specify context, be empathetic, test often, iterate
 - **Accessible:** All our product should be able to be used by as many people as possible. More on that in accessibility.
 - **Joyful:** Doing something good should feel good and be fun.
 - **Responsive:** Our products should work on all devices. Layouts should adapt and support.
+
+## Vision
+
+Our Vision is to create seamless, beautiful and professional design. Accessible for everyone. We want our users to find quickly what they need, have fun while using it and engage highly through them with others to create impact for good.
```

Comment:
@inesczuia I don't think the vision is really helpful here for different reasons:

> Our Vision is to create seamless, beautiful and professional design.
Unclear what seamless, beatiful and professional actually should mean. Sounds a bit random (like something everyone wants) as well.

> Accessible for everyone.
Repeats what is already listed in the principles.

>  We want our users to find quickly what they need, have fun while using it
This is just parts of the definition of UX in other less precise words. 

> engage highly through them
Unclear what that means.